extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
```

Comment:
I don't think these two should be joined as one tagspec.

I'd recommend undoing these changes and instead add a `src` attribute that is not mandatory to the `with json child` tagspec.